src/edu/csus/ecs/pc2/core/model/Problem.java

```diff
@@ -141,6 +141,11 @@ public enum VALIDATOR_TYPE {
     private ClicsValidatorSettings clicsValidatorSettings ;
     private CustomValidatorSettings customValidatorSettings ;
     
+    /**
+     * If true, when loading data sets, load sample data sets before loading other judge's test data sets.
+     */
+    private boolean onLoadDataFilesLoadSamplesFirst = false;
```

Comment:
I'm not sure why the property is called "onLoadDataFilesLoadSamplesFirst".  What is the "on" prefix for?  "loadDataFilesSamplesFirst" is probably better.

Reply:
I agree.

Reply:
pushed fix.